torch/nested/__init__.py

```diff
@@ -410,3 +411,55 @@ def nested_tensor_from_jagged(
 
     return nested_view_from_values_offsets_lengths(
         values, offsets, lengths, ragged_idx=jagged_dim, min_seqlen=min_seqlen, max_seqlen=max_seqlen)
+
+def masked_select(tensor: Tensor, mask: Tensor) -> Tensor:
+    r"""
+    Constructs a nested tensor given a a strided tensor input and a strided mask, the resulting jagged layout Nested Tensor
+    will have values equivalent to running :func:`masked_select` with the provided tensor and mask. The offsets will be 
+    generated from the contiguous True values in the mask.
+
+    Args:
+    tensor (:class:`torch.Tensor`): a strided tensor from which the jagged layout nested tensor is constructed from.
+    mask (:class:`torch.Tensor`): a strided mask tensor which is applied to the tensor input
+
+    Example::
+
+    >>> tensor = torch.randn(3,3)
```

Comment:
I wonder if lint checks code spacing in code comments...
```suggestion
    >>> tensor = torch.randn(3, 3)
```